test/DynamoCoreWpfTests/NodeAutoCompleteSearchTests.cs

```diff
@@ -164,35 +164,82 @@ public void NodeSuggestions_InputPortBuiltInNode_AreCorrect()
+                "FFITarget.FFITarget.DummyCollection.AcceptTypedDictionary",
+                "FFITarget.FFITarget.DummyCollection.ObjectAsArbitraryDimensionArrayImport",
+                "FFITarget.FFITarget.DummyCollection.ReturnArbitraryDictionary",
+                "FFITarget.FFITarget.DummyCollection.ReturnDictionary",
+                "FFITarget.FFITarget.DummyCollection.ReturnDictionaryAsObject",
+                "FFITarget.FFITarget.DummyCollection.ReturnIDictionary",
+                "FFITarget.FFITarget.DummyCollection.ReturnIEnumerable",
+                "FFITarget.FFITarget.DummyCollection.ReturnIEnumerablOfIList",
+                "FFITarget.FFITarget.DummyCollection.ReturnIList",
+                "FFITarget.FFITarget.DummyCollection.ReturnNestedDictionary",
+                "FFITarget.FFITarget.DummyCollection.ReturnNestedIDictionary",
+                "FFITarget.FFITarget.DummyMath.Sum",
+                "FFITarget.FFITarget.HiddenDisposeTracer.GetHiddenDisposer",
+                "FFITarget.FFITarget.TestData.AddItemToFront",
+                "FFITarget.FFITarget.TestData.CreateInternalClass",
+                "FFITarget.FFITarget.TestData.FuncReturningByteAsObject",
+                "FFITarget.FFITarget.TestData.FuncReturningVariousObjectTypes",
+                "FFITarget.FFITarget.TestData.GetDictionary",
+                "FFITarget.FFITarget.TestData.GetHashTable",
+                "FFITarget.FFITarget.TestData.GetIEnumerable",
+                "FFITarget.FFITarget.TestData.GetInt",
+                "FFITarget.FFITarget.TestData.GetMixedObjects",
+                "FFITarget.FFITarget.TestData.GetNestedCollection",
+                "FFITarget.FFITarget.TestData.GetObjectValue",
+                "FFITarget.FFITarget.TestData.GetUlong",
+                "FFITarget.FFITarget.TestData.GetValueFromDictionary",
+                "FFITarget.FFITarget.TestData.GetValueFromHashTable",
+                "FFITarget.FFITarget.TestData.JoinList",
+                "FFITarget.FFITarget.TestData.MultiplyDecimals",
+                "FFITarget.FFITarget.TestData.RemoveItemsAtIndices",
+                "FFITarget.FFITarget.TestData.ReturnNestedDictionary",
+                "FFITarget.FFITarget.TestData.ReturnObject",
+                "FFITarget.FFITarget.TestData.SingleKeyDictionary",
+                "FFITarget.FFITarget.WrapperObject.WrapperGuid",
+                "FFITarget.FFITarget.WrappersTest.CleanedObjects",
+                "FFITarget.System.Attribute.TypeId",
+                "Operators.-",
+                "Operators.!=",
+                "Operators.%",
+                "Operators.&&",
+                "Operators.*",
+                "Operators./",
+                "Operators.||",
+                "Operators.+",
+                "Operators.<",
+                "Operators.<=",
+                "Operators.==",
+                "Operators.>",
+                "Operators.>=",
+                "Operators.Not"
```

Comment:
Why are there so many results?

Reply:
the port thats being matched has changed.

Reply:
`string` type is now skipped

Reply:
I think the first input type is var[]..[] and the second is string, so in both cases, the type is skipped, which should mean 0 results, or am I missing something?

Reply:
Or this should return only the default types, which are around 6 or so.

Reply:
`var[]..[]` is not skipped